modules/pxp-module-puppet

```diff
@@ -26,14 +26,30 @@ end
 
 DEFAULT_EXITCODE = -1
 
-# This method exists to facilitate testing
-def last_child_exit_status
-  return $?
-end
+env_fix_up = nil
+
+def get_env_fix_up
+  # Prepare an environment fix-up to make up for its cleansing performed
+  # by both pxp-agent and the Puppet::Util::Execution.execute function.
```

Comment:
Hmm no, I've tried again and it's there. So probably I just overlooked it. I'll update the comments.